fw.lua

```diff
@@ -890,7 +890,11 @@ function _M.set_option(self, option, value)
 				_fatal_fail("Attempted to set FreeWAF storage zone as " .. tostring(value) .. ", but that lua_shared_dict does not exist")
 			end
 			self._storage_zone = value
-		end
+		end,
+                allowed_content_types = function(value)
```

Comment:
Pedantic- it looks like the spacing is off. Can we properly align this (using tabs)?